### PR TITLE
Add validation losses

### PR DESCRIPTION
**Pull request by sushen25**

### CODE REVIEW

**Review by voirra — APPROVED** [review]

**Review by voirra — APPROVED** [review]

File: alt_train.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                continue
+
+            losses = np.append(losses, loss.item())
+            np.save(path_to_losses_npy_file, losses)
+
+            print('=> Evaluating on validation dataset...')
+            accuracy, test_loss_args = evaluator.evaluate(model)
+            test_loss = _loss(*test_loss_args, length_labels, digits_labels)
+            test_losses = np.append(test_losses, test_loss.item())
+            np.save(path_to_losses_npy_file, test_losses)
```
Comment (sushen25) [inline alt_train.py]: Just a question,
so ur storing both the training and validation losses in the same .npy file?

Comment (sushen25) [reply alt_train.py]: nvm figured it out, it's 2 separate files dw

Comment (voirra) [reply alt_train.py]: actually I found something I missed, hold off from merging

**Comment by voirra** [comment]
rebased onto master, will merge this after I'm happy that the validation losses works

**Pull request merged**